wordpress-mobile/WordPress-iOS
# Update the WordPressShared pod to use the latest version

Fixes #11564. This will update the WPiOS app to use the latest version of WordPressShared, which contains Swift 5 support. Ref. https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/201

To test:
- `bundle exec pod install`
- run unit tests
- run the app

The app should build and run without any issue.

Once the WordPressShared PR is approved, I'll update this PR to the correct version.

Update release notes:
